docs/docs/operator/automatic-release-upgrades.md

```diff
@@ -0,0 +1,55 @@
+# Automatically upgrading the Operator (Experimental)
+
+The Kubescape operator provides an experimental capability to automatically fetch the latest chart version and upgrade the installed release.
+
+## How to Use
+
+To enable this capability, simply simply set the `capabilities.autoUpgrading` to `enable` and configure how often you would like to check for updates by adjusting the cron schedule:
+
+```yaml
+capabilities:
+  autoUpgrading: enable
+
+# [Omitted for brevity...]
+
+# If necessary, adjust when you want the Upgrader to check for updates and apply updates
+helmReleaseUpgrader:
+  schedule: "0 14 * * *"  # Check for updates every day at 14:00
+```
+
+Helm will then install the Kubescape Helm Release Upgrader CronJob and its supporting resources.
+They will keep your Kubescape release up to date.
+
+!!! warning Warning
+    Due to how Helm works, the Helm Release Upgrader requires highly elevated RBAC permissions and leaves orphaned resources even after you uninstall the Kubescape release.
+
+## How it Works
+
+The Auto Upgrading capability is experimental.
+
+It works as follows:
+
+- Creates a ClusterRole with cluster admin privileges, a ServiceAccount and a matching ClusterRoleBinding to have all the permissions necessary to create resources with Helm.
+- Creates a CronJob that updates the local Kubescape Helm repo (`helm repo update`) to fetch the latest chart versions and upgrades the Helm release accordingly (`helm upgrade`).
+- To account for Helm removing and re-creating the release resources during `helm upgrade`, it keeps the RBAC definitions and the CronJob in the cluster with `"helm.sh/resource-policy": keep`.
+
+This means:
+
+- The upgrader runs in a very privileged RBAC context: it can create resources, list Secrets etc.
+- Even after you remove the Kubescape release with `helm -n kubescape uninstall kubescape`, it will still keep the highly privileged RBACs and CronJob in your cluster.
+
+While the CronJob runs in a non-root security context and you can clean up the leftover resources on your own, please evaluate the risks the Release Upgrader introduces against your security needs and threat model.
```

Comment:
The name of the thing needs to stay consistent across the board.
So, it should either be Upgrader; Kubescape Upgrader or Release Upgrader throughout.

Reply:
Fixed.